packages/cli/src/commands/index.ts

```diff
@@ -178,7 +172,10 @@ export function registerCommands(program: CommanderStatic) {
       '--package <name>',
       'Only load config schema that applies to the given package',
     )
-    .option('--lax', 'Do not require environment variables to be set')
+    .option(
+      '--lax',
+      '[DEPRECATED] - Do not require environment variables to be set',
```

Comment:
Thinking about this I don't think we can deprecate this for any of the `config:*` commands. It's useful to be able to print config without supplying all env vars. It would be nice if we could make it not be required anymore, but I feel it needs a proper replacement if you're gonna remove it now